zen_queries/utils.py

```diff
@@ -1,3 +1,9 @@
-def fetch(queryset):
-    queryset._fetch_all()
+from django.db.models.query import QuerySet
+from typing import TypeVar
+
+T = TypeVar("T")
```

Comment:
I attempted to add `bound=QuerySet` here, but that didn't work well with django-stubs as its `QuerySet` definition is generic, and generic types cannot be used as `TypeVar` bounds.

Reply:
In that case, would just using `QuerySet` be better? Currently this method is too generic, and whilst correct for the return type, makes no guarantees inside the method.

Alternatively, if it's generic, shouldn't `def fetch(queryset: QuerySet[T]) -> QuerySet[T]:` work?

Reply:
If we use just `QuerySet`, then a user of django-zen-queries would loose the information about what type of `QuerySet` they're working with when using `fetch()`. Example:

``` python
>>> reveal_type(qs)  # => OrderQuerySet
>>> reveal_type(fetch(qs))  # => QuerySet, but this should still be OrderQuerySet
```

With all the variants of `QueryType[...]` I've attempted, I either get mypy errors or `Any` out.

I think it is more important here that the types help the user, than helping the maintainers of django-zen-queries, as there is a lot less code and churn in the library compared to in an average Django app.